docs/status.md

```diff
@@ -159,3 +159,19 @@ one of the following tracking labels.
 | uniform_quantize         | yes           | revisit      | infeasible     | yes             | no          |
 | while                    | yes           | revisit      | yes            | revisit         | yes         |
 | xor                      | yes           | yes          | yes            | yes             | yes         |
+
+## Note
+
+- **Type inference and Quantization**: `Type Inference` column from the above
```

Comment:
The `Type Inference` column from the table above is intended to focus on non-quantized operations. For the majority of the quantized operations, it is not feasible to infer the result type because the quantization parameters of the result types may vary from those of the operands, with the exception of a few cases that are listed below.